gw_eccentricity/eccDefinition.py

```diff
@@ -1060,21 +1071,85 @@ def interp_extrema(self, extrema_type="pericenters"):
                 " Can not create an interpolant.")
 
     def check_num_extrema(self, extrema, extrema_type="extrema"):
-        """Check number of extrema."""
+        """Check number of extrema.
+
+        Check the number of extrema to determine if there are enough for
+        building the interpolants through the pericenters and apocenters. In
+        cases where the number of extrema is insufficient, i.e., less than 2,
+        we further verify if the provided waveform is long enough to have a
+        sufficient number of extrema.
+
+        If the waveform is long enough, but the chosen method fails to detect
```

Comment:
Here is where you should explain the 5 orbits thing, and if that's really 7 orbits -- but explain this using the variables like num_orbits_to_remove_before_merger, and their default values. Basically, help out the user in figuring out how long the input waveform needs to be to use this option.

Reply:
Done!